components/cdc/src/initializer.rs

```diff
@@ -107,30 +108,9 @@ impl<E: KvEngine> Initializer<E> {
         &mut self,
         change_observer: ChangeObserver,
         cdc_handle: T,
-        concurrency_semaphore: Arc<Semaphore>,
         memory_quota: Arc<MemoryQuota>,
     ) -> Result<()> {
         fail_point!("cdc_before_initialize");
-        let _permit = concurrency_semaphore.acquire().await;
-
-        // When downstream_state is Stopped, it means the corresponding delegate
-        // is stopped. The initialization can be safely canceled.
-        //
-        // Acquiring a permit may take some time, it is possible that
-        // initialization can be canceled.
-        if self.downstream_state.load() == DownstreamState::Stopped {
```

Comment:
shall we also keep the check here? just remove the _permit.

Reply:
It will be checked later soon. So we can remove this pre-check to simplify code.